herbert_core/classes/MPIFramework/@ClusterWrapper/ClusterWrapper.m

```diff
@@ -48,6 +48,12 @@
         % cluster. Property used in debugging to change default framework
         pool_exchange_frmwk_name
     end
+    properties(Dependent,Hidden)
+        % return enviromental variables used by cluster. Normally for testing
+        % It is handle, so it can be used for seting it too.
+        % (Violates class logic so avoid this)
```

Comment:
needs more explanation what class logic violation means here, and what steps should be taken to avoid using it - e.g. if you have an alternative safer means of setting this variable, say what it is.

Also say what class this is, so we can see the handle construction logic.